docs/rules/no-tabs.md

```diff
@@ -0,0 +1,37 @@
+# Disallow tabs in file (no-tabs)
+
+Some people prefer whitespaces instead of tab characters in a file. So they would want to disallow tab anywhere inside a file including comments.
```

Comment:
Tabs are also whitespace. Better to say that some style guides don't allow the use of tab characters anywhere.